invenio_theme_tugraz/views.py

```diff
@@ -10,14 +10,16 @@
 
 from flask import Blueprint, render_template
 from flask_babelex import gettext as _
+from typing import Dict
+from elasticsearch_dsl.utils import AttrDict
 
 from .search import FrontpageRecordsSearch
 
 blueprint = Blueprint(
-    'invenio_theme_tugraz',
+    "invenio_theme_tugraz",
     __name__,
-    template_folder='templates',
-    static_folder='static',
+    template_folder="templates",
```

Comment:
change back the double-qoutes to single-qoutes.
```suggestion
    template_folder='templates',
```